plugins/sonarqube/src/components/SonarQubeCard/SonarQubeCard.tsx

```diff
@@ -232,7 +232,15 @@ export const SonarQubeCard = ({
                 link={value.getComponentMeasuresUrl('COVERAGE')}
                 title="Coverage"
                 leftSlot={<Percentage value={value.metrics.coverage} />}
-                rightSlot={<Value value={`${value.metrics.coverage}%`} />}
+                rightSlot={
+                  <Value
+                    value={
+                      value.metrics.coverage
```

Comment:
Realized this will show 0 coverage as `-`, needs an explicit check for `undefined`

Reply:
Ah, yep. Updated it to cover that scenario.